src/main/java/uk/gov/justice/digital/delius/service/CustodyService.java

```diff
@@ -52,6 +55,7 @@ public CustodyService(
             OffenderManagerService offenderManagerService,
             ContactService contactService) {
```

Comment:
Possibly not one for this PR, but the number of dependencies smells like this class might need breaking up?